types/enzyme/index.d.ts

```diff
@@ -430,7 +430,15 @@ export interface ShallowWrapper<P = {}, S = {}> extends CommonWrapper<P, S> {
      * NOTE: can only be called on wrapper of a single non-DOM component element node.
      * @param [options]
      */
-    dive<P2, S2>(options?: ShallowRendererProps): ShallowWrapper<P2, S2>;
+    dive<P2, S2>(options?: ShallowRendererProps): ShallowWrapper<any, any>;
+
+    /**
+     * Strips out all the not host-nodes from the list of nodes
+     *
+     * This method is useful if you want to check for the presence of host nodes
+     * (actually rendered HTML elements) ignoring the React nodes.
+     */
+    hostNodes(): ShallowWrapper<any, any>;
```

Comment:
These can be `Shallow|ReactWrapper<HTMLAttributes>` for better completions on calls chained after this

Reply:
this method is in the ShallowWrapper interface, therefore should it only be `ShallowWrapper<HTMLAttributes, any>`?

Reply:
Right, yeah, was just being lazy, sorry. `ShallowWrapper<HTMLAttributes>` here and `ReactWrapper<HTMLAttributes>` below. The reason for omitting any is that the second parameter is state, and host nodes can't have state. So the default of `{}` handles shutting down access to `wrapper.state().x`.

Reply:
Ah alright, thanks for the info. will push the changes